planner/core/exhaust_physical_plans.go

```diff
@@ -1555,9 +1555,26 @@ func (la *LogicalApply) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([
 		return nil, true
 	}
 	join := la.GetHashJoin(prop)
+	var columns []*expression.Column
+	for _, colColumn := range la.CorCols {
+		columns = append(columns, &colColumn.Column)
+	}
+	count := 1.0
+	if la.stats.RowCount != 0 {
+		count = getCardinality(columns, la.schema, la.stats) / la.stats.RowCount
```

Comment:
What does this mean?

Reply:
It used to calculate the `NDV` of the correlated columns. I will add some comments.